src/open_inwoner/pdc/tests/test_data_migrations.py

```diff
@@ -1,3 +1,5 @@
+from unittest import skip
+
```

Comment:
I'd recomment to remove instead of skip these tests if they're no longer necessary